src/ResourceManager/HDInsight/Commands.HDInsight/ManagementCommands/DisableAzureHDInsightOMSCommand.cs

```diff
@@ -0,0 +1,55 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.HDInsight.Commands;
+using Microsoft.Azure.Management.HDInsight.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.HDInsight
+{
+    [Cmdlet(VerbsLifecycle.Disable, Constants.CommandNames.AzureHDInsightOMS)]
+    [Alias("Disable-AzureRmHDInsightOMS")]
+    [OutputType(typeof(OperationResource))]
+    public class DisableAzureHDInsightOMSCommand : HDInsightCmdletBase
+    {
+        #region Input Parameter Definitions
+
+        [Parameter(
+            Position = 0,
+            Mandatory = true,
+            HelpMessage = "Gets or sets the name of the cluster to disable Operations Management Suite(OMS).",
+            ValueFromPipelineByPropertyName = true,
+            ValueFromPipeline = true)]
```

Comment:
@shefaliv ValueFromPipeline should add value.
Can you point me a scenario where you will need ValueFromPipeline that cannot be done with ValueFromPipelineByPropertyName.
Usually you use ValueFromPipeline when you need to pass in the entire object, here you have 2 string properties and ValueFromPipelineByPropertyName will do it.
So I agree consistency, but continuing to do something something that is not right to begin with is we need to avoid.